src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -20,6 +20,16 @@ object AgoraEntity {
       case HttpEntity.Empty ⇒ new AgoraEntity()
     }
 
+  implicit val SeqAgoraEntityUnmarshaller =
+    Unmarshaller[Seq[AgoraEntity]](`application/json`) {
+      case HttpEntity.NonEmpty(contentType, data) => {
+        grater[AgoraEntity].fromJSONArray(data.asString)
+      }
+      case HttpEntity.Empty => {
+        Seq(new AgoraEntity())
```

Comment:
But then would it be weird to have a findSingle method that returns a Seq[AgoraEntity]?